tsdoc/schemas/tsdoc.schema.json

```
@@ -29,6 +29,20 @@
       }
     },
 
+    "supportedHtmlElements": {
+      "description": "The HTML element names that are supported in this configuration. Used in conjunction with the \"reportUnsupportedHtmlElements\" setting.",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^[a-zA-Z0-9-]+$"
+      }
+    },
+
+    "reportUnsupportedHtmlElements": {
+      "description": "Whether an error should be reported when an unsupported HTML element is encountered in a doc comment. Defaults to \"true\" if \"supportedHtmlElements\" contains at least one item, \"false\" if not.",
```

```suggestion
      "description": "Whether an error should be reported when an unsupported HTML element is encountered in a doc comment. The default value is \"false\".",
```

👆 Just to confirm, this is the design that we settled on, right?

---

The default value is `false`, unless `supportedHtmlElements` is present. How's this sound?
```
"Whether an error should be reported when an unsupported HTML element is encountered in a doc comment. Defaults to \"true\" if the \"supportedHtmlElements\" field is present, \"false\" if not.",
```